File: Java/1012.java
package Java;

import java.util.Scanner;

public class Uri1012 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        Float a = sc.nextFloat();
        Float b = sc.nextFloat();
        Float c = sc.nextFloat();
        System.out.printf("TRIANGULO: %.3f\n", (a * c) / 2);
        System.out.printf("CIRCULO: %.3f\n", 3.14159 * c * c);
        System.out.printf("TRAPEZIO: %.3f\n", ((a + b) * c) / 2);
        System.out.printf("QUADRADO: %.3f\n", b * b);
        System.out.printf("RETANGULO: %.3f\n", a * b);
    }

}

File: Kotlin/1012.kt
import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`);
    val pi = 3.14159;
    val a = input.nextDouble();
    val b = input.nextDouble();
    val c = input.nextDouble();
    println("TRIANGULO: %.3f".format((a * c) / 2));
    println("CIRCULO: %.3f".format(pi * c * c));
    println("TRAPEZIO: %.3f".format((a + b) * c / 2));
    println("QUADRADO: %.3f".format(b * b));
    println("RETANGULO: %.3f".format(a * b));
}